content/en/database_monitoring/setup_mysql/aurora.md

```diff
@@ -36,7 +36,7 @@ Performance impact
 Database Monitoring runs as an integration on top of the base Agent ([see benchmarks][1]).
 
 Proxies, load balancers, and connection poolers
-: The Agent must connect directly to the host being monitored, preferably through the instance endpoint. The Agent should not connect to the database through a proxy, load balancer, connection pooler, or the **Aurora cluster endpoint**. While this can be an anti-pattern for client applications, each Agent must have knowledge of the underlying hostname and should stick to a single host for its lifetime, even in cases of failover. If the Datadog Agent connects to different hosts while it is running, the values of metrics will be incorrect.
+: The Agent must connect directly to the host being monitored, preferably through the instance endpoint. The Agent should not connect to the database through a proxy, load balancer, connection pooler, or the **Aurora cluster endpoint**. The Agent relies on the statistics table of each instance, so each Agent must have knowledge of the underlying hostname and should stick to a single host for its lifetime, even in cases of failover. If the Datadog Agent connects to different hosts while it is running, the values of metrics will be incorrect.
```

Comment:
thinking about it more I think "stats table" sounds a bit clearer here, this phrasing is what we use in customer conversations most often

Reply:
I think this is good, but if we're revamping, might be a good time to improve the overall message? We wrote this a long time ago, but I think we can be better about describing the tradeoffs of connecting to the cluster endpoint.

The key takeaways:
* if you connect to the cluster endpoint, the Agent will connect to one _random_ replica and only provide visibility to that replica. Most of the time this is not what customers want
* if there is a failover event, the Agent will calculate the difference in statistics between two different hosts, which can produce wildly incorrect stats
* for these reasons, we recommend using the instance endpoint